tokens/src/components/Navbar.json

```diff
@@ -160,23 +85,99 @@
             }]
           }
         },
-        "icon-bg": {
-          "value": "url(\"data:image/svg+xml,%3csvg viewBox='0 0 30 30' xmlns='http://www.w3.org/2000/svg'%3e%3cpath stroke='{navbar.light.color.text}' stroke-width='2' stroke-linecap='round' stroke-miterlimit='10' d='M4 7h22M4 15h22M4 23h22'/%3e%3c/svg%3e\")",
-          "source": "$navbar-light-toggler-icon-bg",
-          "outputReferences": false,
-          "modify": [{
-            "type": "str-replace",
-            "toReplace": "#",
-            "replaceWith": "%23"
-          }]
+        "brand": {
+          "base": { "value": "{navbar.light.color.active.value}", "source": "$navbar-light-brand-color" },
+          "hover": { "value": "{navbar.light.color.active.value}", "source": "$navbar-light-brand-hover-color" }
+        }
+      }
+    }
+  },
+  "spacing": {
+    "navbar": {
+      "padding": {
+        "y": { "value": "calc({spacer.base.value} / 2)", "source": "$navbar-padding-y" },
+        "x": {
+          "base": { "value": "{spacer.base.value}", "source": "$navbar-padding-x" },
+          "nav-link": { "value": ".5rem", "source": "$navbar-nav-link-padding-x" }
         }
       },
       "brand": {
-        "color": {
-          "base": { "value": "{navbar.light.color.active.value}", "source": "$navbar-light-brand-color" },
-          "hover": {
-            "value": "{navbar.light.color.active.value}",
-            "source": "$navbar-light-brand-hover-color"
+        "padding": {
+          "y": {
+            "value": "calc(({navbar.nav.link.height.value} - {navbar.brand.height.value}) / 2)",
+            "source": "$navbar-brand-padding-y"
+          }
+        }
+      },
+      "toggler": {
+        "padding": {
+          "y": { "value": ".25rem", "source": "$navbar-toggler-padding-y" },
+          "x": { "value": ".75rem", "source": "$navbar-toggler-padding-x" }
+        }
+      }
+    }
+  },
+  "typography": {
+    "navbar": {
+      "brand": {
+        "font-size": { "value": "{font.size.lg.value}", "source": "$navbar-brand-font-size" }
+      },
+      "nav-link": {
+        "height": {
+          "value": "calc({font.size.base.value} * {line-height.base.value} + .5rem * 2)",
+          "source": "$nav-link-height"
+        }
+      },
+      "toggler": {
+        "font-size": { "value": "{font.size.lg.value}", "source": "$navbar-toggler-font-size" }
+      }
+    }
+  },
+  "size": {
+    "navbar": {
+      "nav": {
+        "scroll": {
+          "max-height": { "value": "75vh", "source": "$navbar-nav-scroll-max-height" }
+        }
+      },
+      "brand": {
+        "height": {
+          "value": "calc({navbar.brand.font.size.value} * {line-height.base.value})",
+          "source": "$navbar-brand-height"
+        }
+      },
+      "toggler": {
+        "border-radius": {
+          "value": "{btn.border.radius.base.value}", "source": "$navbar-toggler-border-radius"
+        }
+      }
+    }
+  },
+  "other": {
```

Comment:
maybe use `icons` key here or some other? because it only contains svg icons as values, we should probably come up with some specific category, not just `other`

Reply:
In Style Dictionary docs I found the `content` category. I think this is good for we)